backend/src/infrastructure/Data/TodoService.cs

```diff
@@ -14,49 +17,239 @@ public TodoService(ApplicationDbContext context)
         _context = context;
     }
 
-    public IEnumerable<TodoList> GetLists(string userId)
+    public IEnumerable<TodoList> GetLists(Guid user)
     {
         return _context.TodoLists.AsNoTracking()
-                                 .Where(list => list.Owner.Equals(Guid.Parse(userId)))
-                                 .Select(list => new TodoList { Id = list.Id, Name = list.Name, Owner = list.Owner })
+                                 .Where(list => list.Owner.Equals(user))
+                                 .Select(list => new TodoList
+                                 {
+                                     Id = list.Id,
+                                     Name = list.Name,
+                                     Owner = list.Owner,
+                                     Items = Enumerable.Empty<TodoItem>()

```

Comment:
I've got an error because of this line.
Error: 

> System.InvalidOperationException: The client projection contains a reference to a constant expression of 'System.Linq.EmptyPartition<core.TodoItem>'. This could potentially cause a memory leak; consider assigning this constant to a local variable and using the variable in the query instead.


If this Empty item is moved to the local variable as the error solution is proposing then it is working correctly.